src/components/SoftKey/SoftKey.jsx

```diff
@@ -33,66 +31,61 @@ const Button = props => {
       onClick={handleButtonClick}
       onFocus={handleCheckFocus}
     >
-      {props.icon ? <i class={props.icon} /> : null}
+      {props.icon ? <span className={props.icon} /> : null}
```

Comment:
This seems to be the only relevant change. We shouldn't mix styling changes with fixes or new features, especially not if this changes are not consistent with the existing code base.

Reply:
My bad, linter did it automatically